subprojects/core/src/main/java/org/gradle/api/tasks/JavaExec.java

```diff
@@ -337,9 +372,16 @@ public FileCollection getClasspath() {
      */
     public JavaExec copyTo(JavaForkOptions options) {
         javaExecHandleBuilder.copyTo(options);
+        copyJvmArgProvidersTo(options);
         return this;
     }
 
+    private void copyJvmArgProvidersTo(JavaForkOptions options) {
```

Comment:
Can be `static`, but how about inlining it?

Reply:
I am using it in two places. Are you saying it is not worth the extra method and I should repeat the code?